web/libs/editor/src/tags/object/RichText/view.jsx

```diff
@@ -587,7 +587,7 @@ class RichTextPieceView extends Component {
           key="root"
           className={cn("richtext").elem("iframe").mix("htx-richtext").toClassName()}
           referrerPolicy="no-referrer"
-          sandbox="allow-same-origin allow-scripts"
+          sandbox="allow-same-origin allow-scripts allow-popups allow-popups-to-escape-sandbox allow-presentation"
```

Comment:
You use `inline="true"` in the video and it actually makes sense to not have stacked iframes. Also this attribute applies to top-level iframe not the youtube one.
I'd recommend to check youtube iframes with both values for `inline` without changes to `sandbox` param and if it works then we can roll back changes here. Check fullscreen in youtube as well!
And if it requires this change to `sandbox` then we can leave it here.